components/director/pkg/graphql/schema.graphql

```diff
@@ -506,6 +512,8 @@ type Mutation {
 	updateAPI(id: ID!, in: APIDefinitionInput!): APIDefinition! @hasScopes(path: "mutation.updateAPI")
 	deleteAPI(id: ID!): APIDefinition @hasScopes(path: "mutation.deleteAPI")
 	refetchAPISpec(apiID: ID!): APISpec @hasScopes(path: "mutation.refetchAPISpec")
+	generateOneTimeTokenForRuntime(id: ID!): OneTimeToken! @hasScopes(path: "mutation.generateOneTimeTokenForRuntime")
```

Comment:
define scopes in `scopes.yaml` file, IMO it can be `application:write` and `runtime:write`